setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = chemdyg
-version = 0.1.2
+version = 0.1.3
```

Comment:
The version update should be done in a separate PR.

Reply:
Correct back to v0.1.2